org.easyrpg.player.yml

```diff
@@ -0,0 +1,97 @@
+app-id: org.easyrpg.player
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: easyrpg-player
+rename-desktop-file: easyrpg-player.desktop
+rename-appdata-file: easyrpg-player.metainfo.xml
+rename-icon: easyrpg-player
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --filesystem=home # user can store games anywhere
+  - --device=all # needed for gamepad access
+modules:
+  - name: liblcf
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+      - -DLIBLCF_ENABLE_TOOLS=OFF
+      - -DLIBLCF_WITH_XML=OFF
+      - -DLIBLCF_ENABLE_TESTS=OFF
+      - -DLIBLCF_UPDATE_MIMEDB=OFF
+    sources:
+      - type: git
+        url: https://github.com/EasyRPG/liblcf.git
+        tag: '0.8'
+        commit: 9c09529e3de0a4106470082ef48a00d4c3593c33
+    cleanup:
+      - /include
+      - /lib/pkgconfig
+      - /lib/cmake
+      - /share
+  - name: libfmt
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+      - -DFMT_TEST=OFF
+      - -DBUILD_SHARED_LIBS=TRUE
+    sources:
+      - type: git
+        url: https://github.com/fmtlib/fmt.git
+        tag: '9.1.0'
+        commit: a33701196adfad74917046096bf5a2aa0ab0bb50
+    cleanup:
+      - /include
+      - /lib/pkgconfig
+      - /lib/cmake
+  # this is copied from shared-modules to not add the dependency on it
+  - name: libfluidsynth
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+    sources:
+      - type: git
+        url: https://github.com/FluidSynth/fluidsynth.git
+        tag: v2.3.4
+        commit: 5ecdc4568e45123216c6888892caad07918ef127
+    cleanup:
+      - /bin
+      - /include
+      - /lib/cmake
+      - /lib/pkgconfig
+      - /share
+      - '*.so'
+  - name: libxmp
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DBUILD_STATIC=OFF
+      - -DLIBXMP_DISABLE_DEPACKERS=ON
+      - -DLIBXMP_DISABLE_PROWIZARD=ON
+    sources:
+      - type: git
+        url: https://github.com/libxmp/libxmp.git
+        tag: libxmp-4.6.0
+        commit: 8201d26cf933688a8be64292457c429fd8e654ab
+    cleanup:
+      - /include
+      - /lib/cmake
+      - /lib/pkgconfig
+      - '*.so'
+  - name: player
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+    sources:
+      - type: git
+        url: https://github.com/EasyRPG/Player.git
+        tag: '0.8'
+        commit: f8e41f43b619413f95847536412b56f85307d378
+    cleanup:
+      - /share/bash-completion
+      - /share/pixmaps
+    post-install:
+      # Fixup for app id (since rename-* options leave this out)
+      - sed -i 's|easyrpg-player.desktop|org.easyrpg.player.desktop|' /app/share/metainfo/easyrpg-player.metainfo.xml
```

Comment:
1. don't use sed for that.
2. this is actually taken care of by flatpak-builder when calling `rename-desktop-file`

Reply:
Okay, will remove. It did not work while testing, hence the comment.